### PR TITLE
chore: remove rinkeby networks

### PR DESCRIPTION
**Pull request by bartlomiej94**
Closes https://github.com/OffchainLabs/arbitrum-sdk/issues/231.

### CODE REVIEW

**Comment by bartlomiej94** [comment]
@DZGoldman what did you have in mind in terms of the [special deprecation order](https://github.com/OffchainLabs/arbitrum-sdk/issues/231#issue-1506785304)?

File: src/lib/dataEntities/networks.ts [hunk 1]
```diff
@@ -169,15 +169,6 @@ export const l1Networks: L1Networks = {
     isCustom: false,
     isArbitrum: false,
   },
-  5: {
```
Comment (yahgwai) [inline src/lib/dataEntities/networks.ts:172]: Should this be removing rinkeby instead?

**Review by yahgwai — COMMENTED** [review]
Can we also remove the l2 rinkeby network?

File: src/lib/dataEntities/networks.ts [hunk 2]
```diff
@@ -198,20 +189,6 @@ export const l2Networks: L2Networks = {
...
     depositTimeout: 888000,
   },
-  421611: {
-    chainID: 421611,
-    name: 'ArbRinkeby',
-    explorerUrl: 'https://testnet.arbiscan.io',
-    partnerChainID: 4,
-    isArbitrum: true,
-    tokenBridge: rinkebyTokenBridge,
-    ethBridge: rinkebyETHBridge,
```
Comment (yahgwai) [inline src/lib/dataEntities/networks.ts:208]: I think the `rinkebyETHBridge` and `rinkebyTokenBridge` arent referenced now

**Comment by bartlomiej94** [comment]
@DZGoldman also mentioned adding a deprecation order for Rinkeby, what would that refer to?

**Review by yahgwai — APPROVED** [review]

**Comment by yahgwai** [comment]
It looks like there's also a small change to make to the readme, and some stuff to remove from test files/scripts - do a global search for "rinkeby"

**Review by spsjvc — APPROVED** [review]
LGTM, I've also addressed @yahgwai's comments

**Pull request merged**